photo-helper/src/main/java/io/intrepid/photohelper/PhotoHelper.java

```diff
@@ -46,8 +48,51 @@
      * @param view               - The {@link io.intrepid.photohelper.PhotoContract.View View} interface (may very likely be the {@link Fragment})
      * @param removeUnusedPhotos - If true, delete unused new photos that the user takes with the camera
      */
-    public PhotoHelper(@NonNull Fragment fragment, @NonNull PhotoContract.View view, boolean removeUnusedPhotos) {
-        this.fragment = fragment;
+    public PhotoHelper(@NonNull final Fragment fragment, @NonNull PhotoContract.View view, boolean removeUnusedPhotos) {
+        this.contextDelegate = new ContextDelegate() {
+            @Override
+            public void requestPermissions(@NonNull String[] permissions, int requestCode) {
+                fragment.requestPermissions(permissions, requestCode);
+            }
+
+            @Override
+            public ContextWrapper getContext() {
+                return fragment.getActivity();
+            }
+
+            @Override
+            public void startActivityForResult(Intent intent, int requestCode) {
+                fragment.startActivityForResult(intent, requestCode);
+            }
+        };
+        this.view = view;
+        this.removeUnusedPhotos = removeUnusedPhotos;
+    }
+
+    /**
+     * Creates a PhotoHelper instance
+     *
+     * @param activity           - The {@link Activity} that will be calling the {@link PhotoHelper}
+     * @param view               - The {@link io.intrepid.photohelper.PhotoContract.View View} interface (may very likely be the {@link Fragment})
+     * @param removeUnusedPhotos - If true, delete unused new photos that the user takes with the camera
+     */
+    public PhotoHelper(@NonNull final Activity activity, @NonNull PhotoContract.View view, boolean removeUnusedPhotos) {
+        this.contextDelegate = new ContextDelegate() {
+            @Override
+            public void requestPermissions(@NonNull String[] permissions, int requestCode) {
+                ActivityCompat.requestPermissions(activity, permissions, requestCode);
+            }
+
+            @Override
+            public ContextWrapper getContext() {
```

Comment:
What's the reason behind returning ContextWrapper instead of just Context?  Also, can you add a @NonNull annotation to the return variable?  If we can't guarantee it is non-null, then this should have a @Nullable annotation.